src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,21 @@
+.catalog {
+  box-sizing: border-box;
+  display: flex;
+  padding: 50px;
+  margin: 0 auto;
+  width: 1050px;
+  height: 962px;
```

Comment:
you shouldn't set 'magic' width and height for block that can contain more cards etc,